vector/v.out.ogr/testsuite/test_v_out_ogr.py

```diff
@@ -44,15 +55,27 @@ def setUpClass(cls):
 
     @classmethod
     def tearDownClass(cls):
-        """Remove the temporary region"""
+        """!Remove the temporary region"""
```

Comment:
```suggestion
        """Remove the temporary region"""
```

The exclamation marks were useful back when we were using Doxygen for Python doc, but not anymore.